<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentResearchCreationDialog</class>
 <widget class="QDialog" name="CurrentResearchCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Исследование текущей детали</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveFolderPathLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>46</y>
     <width>371</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseSaveFolderPathPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>46</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveFolderPathLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Папка для сохранения</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveResearchToDiskCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить исследование на диск</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="imageSourcePhotoShootingOnlyCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>616</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>В качестве получения изображений использовать только фотосъёмку</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="creationDateTimeLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>13</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата и время создания</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="creationDateTimeEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="useMachineForDetailMovementCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>616</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Использовать станок для помещения детали в нужную позицию ракурса</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>saveResearchToDiskCheckBox</tabstop>
  <tabstop>creationDateTimeEdit</tabstop>
  <tabstop>saveFolderPathLineEdit</tabstop>
  <tabstop>chooseSaveFolderPathPushButton</tabstop>
  <tabstop>imageSourcePhotoShootingOnlyCheckBox</tabstop>
  <tabstop>useMachineForDetailMovementCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CurrentResearchCreationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CurrentResearchCreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
